addons/website_slides/static/src/js/slides_category_add.js

```diff
@@ -29,18 +33,23 @@ var CategoryAddDialog = Dialog.extend({
     },
 
     //--------------------------------------------------------------------------
-    // Private
+    // Handlers
     //--------------------------------------------------------------------------
 
-    _formValidate: function ($form) {
-        $form.addClass('was-validated');
-        return $form[0].checkValidity();
-    },
-
-    _onClickFormSubmit: function (ev) {
-        var $form = this.$('#slide_category_add_form');
-        if (this._formValidate($form)) {
-            $form.submit();
+    /**
+     * Handle form submission on click button or press ENTER key. Prevent multiple submissions
+     * @private
+     */
+    _onFormSubmit: function (ev) {
```

Comment:
We usually describe params in JS docstring, so we could add `@param {Event} ev` for good measures.